### ISSUE TITLE
[Feature] AddStatusBarClass (multiple HUDs)

### ISSUE THREAD

**Issue by jekyllgrim** [ZDoom/gzdoom#2585]
### Which game are you running with GZDoom?

Doom 2

### What Operating System are you using?

Windows 10

### Is your feature request related to a problem? Please describe.

HUD mods are pretty popular nowadays, and being limited to just one HUD is annoying, especially in cases when, for example, a map-focused project comes with its own HUD for very minimal reasons. It would be really great if `AddStatusbarClass` existed and allowed loading multiple HUDs that the player would be able to cycle through dynamically.

### Describe the solution you'd like

`AddStatusBarClass` as an extension of the existing `StatusBarClass`
Option to cycle through loaded HUDs in HUD Options
*Possibly* a way to draw multiple HUDs over each other at the same time

### Describe alternatives you've considered

Currently the only alternative is to create an abomination through RenderOvelray

**Comment by MajorCooke** [comment]
Agreed. If there is multiple ways to do this without having to resorting to using harder functionalities and calculating everything with the Screen.Draw functions, that'd be appreciated.
